src/reporters/json.ts

```diff
@@ -380,11 +380,14 @@ export default class JSONReporter extends BaseReporter {
           journey,
           step,
           url,
+          root_fields: {
+            duration: {
```

Comment:
Can we nest this one level deeper in `step.duration.us`? That will make it more clear to users.

Reply:
Hmmm thinking a bit more, can we keep it in top level and not add it under root fields if thats the case.

Reply:
Sure, we'll need to add it to `synthtypes.go` in heartbeat, but that's not a big deal.